### DISCUSSION TITLE
Refused to load the image 'data:,iVBORw0KGgo=' because it violates the following Content Security Policy directive: "img-src 'self' https://b.stripecdn.com/ https://files.stripe.com/".

### DISCUSSION THREAD

**Discussion by anurudha-singh** [flutter-stripe/flutter_stripe#1483] [Q&A]
We are currently encountering a critical issue while integrating Stripe Checkout Session for payments into our Flutter application. We have a mobile app which is using flutter and stripe checkout session for payment. We did our testing with the developer option on stripe and it was working perfectly but when we switched to the live account we are facing this issue [**_**Refused to load the image 'data:,iVBORw0KGgo=' because it violates the following Content Security Policy directive: "img-src 'self' https://b.stripecdn.com/ https://files.stripe.com/".**_**]. Our App relies on this integration to collect payments from the users and it not working is creating a huge blocker for us. We would really appreciate if you can provide immediate attention to this issue and let us know what we can do to fix this issue.

**Comment by jonasbark** [comment]
Please contact Stripe Support about this

**Reply by anurudha-singh** [reply]
We have already connected with them and they had suggested us to create a new issue on Github. Now, Please suggest something else to resolve this issue.

**Reply by anurudha-singh** [reply]
Hello team,

This is an urgent issue and there are many things depend on this one. So, please check it once again at your end and let us know at the earliest. Like I mentioned yesterday, we already had conversation with Stripe, they suggested to contact you here. Please suggest.
